ext/winevt/winevt_c.h

```diff
@@ -70,6 +78,7 @@ struct WinevtQuery
   LONG timeout;
   BOOL renderAsXML;
   BOOL preserveQualifiers;
+  LocaleInfo localeInfo;
```

Comment:
Ditto

```diff
-  LocaleInfo localeInfo;
+  LocaleInfo *localeInfo;
```

Reply:
Changing to use pointer causes SEGV.... :cry:

Reply:
Of course you have to change also related codes. e.g.)

```diff
- localeInfo.langID
+ localeInfo->langID
```

```diff
- localeInfo = default_locale;
+ localeInfo = &default_locale;
```